packages/frontend/src/features/sqlRunner/hooks/useTables.tsx

```diff
@@ -104,14 +104,16 @@ export const useRefreshTables = ({
 }: Pick<GetTablesParams, 'projectUuid'>) => {
     const queryClient = useQueryClient();
 
-    return useMutation({
-        mutationFn: () => refreshTables({ projectUuid }),
-        onSuccess: async () => {
-            await queryClient.invalidateQueries([
-                'sqlRunner',
-                'tables',
-                projectUuid,
-            ]);
+    return useMutation<ApiWarehouseCatalog, { message: string }, void>(
```

Comment:
It's most likely the error will be of type `ApiError`
```suggestion
     return useMutation<ApiWarehouseCatalog, ApiError>(
```